app/controllers/characters_controller.rb

```diff
@@ -24,26 +24,32 @@ def show
     @character = Character.find(params[:id])
   end
 
+  def edit
+    @character = Character.find(params[:id])
+  end
+
   def update
-    @character = Character.new(params[:id])
+    @character = Character.find(params[:id])
     if @character.update(character_params)
       flash[:notice] = "Character Added!"
+      redirect_to games_path
     else
       flash[:notice] = @character.errors.full_messages.join(" ")
+      render 'show'
     end
   end
 
   protected
 
   def character_params
-    params.require(:character).permit(
-                                        :name,
-                                        :user_id,
-                                        :game_id,
-                                        :birthday,
-                                        :hobby_1,
-                                        :hobby_2
-                                        )
+  params.require(:character).permit(
+                                      :name,
```

Comment:
Align the parameters of a method call if they span more than one line.<br>Indent the first parameter one step more than the previous line.